jekyll/_cci2/deploy-to-amazon-sagemaker.adoc

```diff
@@ -273,18 +273,18 @@ Let's break down the `deploy-model-through-to-prod workflow`. Our first job is `
 
```

Comment:
Line above I think `deploy-model-through-to-prod workflow` should be `deploy-model-through-to-prod` workflow